`import!` macro in submodules is broken

Since https://github.com/microsoft/winrt-rs/pull/174 I can't get my import to work any more.

I previously had the following code, which worked:
```rust
winrt::import!(
    dependencies
        os
    modules
        "windows.foundation"
        "windows.devices.enumeration"
        "windows.devices.midi"
        "windows.storage.streams"
);
```


Now I changed it to this:
```rust
winrt::import!(
    dependencies
        os
    types
        windows::foundation::*
        windows::devices::enumeration::*
        windows::devices::midi::*
        windows::storage::streams::*
);
```

(It doesn't work when I specify individual types instead of `*` either.)

I get the following error:

```
error[E0433]: failed to resolve: unresolved import
  --> src\backend\winrt\mod.rs:10:1
   |
10 | / winrt::import!(
11 | |     dependencies
12 | |         os
13 | |     types
...  |
20 | |         windows::storage::streams::*
21 | | );
   | |  ^
   | |  |
   | |__unresolved import
   |    help: a similar path exists: `std::os::windows`
```

... which is not very helpful. Have you seen this before? I'm even getting the same error when I import *only* `windows::foundation::*`, so it looks like at has to do something with the way that I have the import in my codebase (which you can find [here](https://github.com/Boddlnagg/midir/tree/winrt-rs-broken) and build with `cargo build --feature winrt`).

--- a/crates/winmd/src/types/debug.rs
+++ b/crates/winmd/src/types/debug.rs
@@ -16,10 +16,14 @@ pub fn debug_tokens(type_name: &TypeName, interfaces: &Vec<RequiredInterface>) -
         type_name.name == "IStringable" && type_name.namespace == "Windows.Foundation";
 
     let implementation = if implements_istringable && !is_istringable {
+        let istringable_namespace = crate::types::namespace::to_namespace_tokens(
+            "Windows.Foundation",
+            &type_name.namespace,
+        );
         quote! {
             "{:?}",
             {
-                let s: crate::windows::foundation::IStringable = self.into();
+                let s: #istringable_namespace IStringable = self.into();
                 s
             }
         }

